### DISCUSSION TITLE
Feel like an idiot... What's my ID?

### DISCUSSION THREAD

**Discussion by Mastiffen** [ESPresense/ESPresense#222] [Q&A]
I have been checking out the MQTT messages coming from the ESP, and I'm pretty sure this is my phone (an Huwaei P30 Pro):

`    "id": "msft:cdp:0902",
    "disc": "fb0f456073d7e8b0bdd70c261055345916a5c942",
    "idType": 12,
    "rssi@1m": -71,
    "rssi": -70,
    "mac": "4d95d1e1dd24",
    "raw": 0.94,
    "distance": 0.97,
    "speed": 0
`

 But I can't get the sensor in Home Assistant to show that it's there. I have tried to use the id, and to use the disc (whatever that is), and even the ID and disc with a dash or a colon between them, but none of them seem to be correct, I keep getting "not home" in the sensor Tor (my first name isn't really Mastiff, and not even Mastiffen, which I use because Mastiff was taken...). Can somebody please put me out of my misery and tell me what I should use as the id in the config.yaml?

**Comment by Mastiffen** [comment]
I found the solution. Turned out not to be the phone, and my Huawei was killing the Hass app even though I have done away with battery optimization on it. Using Beacon Scope fixed that.

**Reply by DTTerastar** [reply]
msft:cdp:0902 is actually a microsoft windows laptop.  We can't tell them apart, but we can count the number in a room!

**Comment by Mastiffen** [comment]
Ah, it's my Surface Pro! I see!
